src/main/resources/view/QuizQuetionListCard.fxml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The file name `QuizQuetionListCard.fxml` is wrongly spelt and should be spelt `QuizQuestionListCard.fxml`.